backend/editor/entries.py

```diff
@@ -101,4 +186,40 @@ def update_nodes(label, entry, incomingData):
 
     params = dict(incomingData, id=entry)
     result = session.run(" ".join(query), params)
-    return result
\ No newline at end of file
+    return result
+
+def update_node_children(entry, incomingData):
+    """
+    Helper function used for updation of node children with given id 
+    """
+    # Parse node ids from Neo4j Record object
+    current_children = [record["b.id"] for record in list(get_children(entry))]

```

Comment:
What is this `b.id` parameter ? We should rename it `child.id`.

It's a change required in `get_children`.

It does not seems a big deal, but really naming variables and parameters rightly is very important in the long run.